src/components/common/ui/form/numbered-text-area-mui/utils/index.tsx

```diff
@@ -0,0 +1,13 @@
+export const transformValue = (value: string): string => {
+  if (value.includes(',') || value.includes(';')) {
+    value = value
+      .replace(/,/g, '\n')
+      .replace(/;/g, '\n')
+      .replace(/\s+/g, '\n')
```

Comment:
you don't need to repeat `replace(/\s+/g, '\n')` and `replace(/\n\n/g, '\n')` because in any case you will do these steps after if statement.
Moreover, you can write like:
` value = value.replace(/\s+/g, '\n').replace(/\n\n/g, '\n');`

Reply:
![image](https://github.com/fictadvisor/fictadvisor-web/assets/89925253/d06725ef-ec72-4313-9a93-d6890a27bf67)